src/igvfd/schemas/changelogs/gene.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`gene.json`*
 
+### Minor changes since schema version 3
+
+* Add `GRCM39` to `dependentSchemas` and `locations` assemblies
```

Comment:
there is no need to mention "dependent schemas". It is just logic of dependencies, and I don't think should be included in the changelog.

Reply:
okay i'll remove it